### PR TITLE
Added docstrings for :class:`~.ValueTracker`

### PR DESCRIPTION
**Pull request by kolibril13**
## Changelog / Overview

Added docs for ValueTracker and added the function  __isub__, because there was already __iadd__

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [x] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [x] The PR title is descriptive enough
- [x] The PR is labeled correctly
- [x] The changelog entry is completed if necessary
- [x] Newly added functions/classes either have a docstring or are private
- [x] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by kolibril13** [comment]
See here: https://manimce--1246.org.readthedocs.build/en/1246/reference/manim.mobject.value_tracker.ValueTracker.html?highlight=valuetracker#manim.mobject.value_tracker.ValueTracker

File: manim/mobject/value_tracker.py [hunk 1]
```diff
@@ -37,8 +37,14 @@ def construct(self):
...
                 self.play(pointer_value.animate.set_value(5)),
-                self.wait()
+                self.wait(0.5)
                 self.play(pointer_value.animate.set_value(3))
+                self.play(pointer_value.animate.increment_value(-2))
+                self.wait(0.5)
+                pointer_value += 1.5
+                self.wait(1)
+                pointer_value -= 4
+                self.wait(0.5)
```
Comment (RickyC0626) [inline manim/mobject/value_tracker.py:47]: I think increasing the wait times would make the animation loop more smoothly.
```py
self.play(pointer_value.animate.increment_value(-2))
self.wait(1)
pointer_value += 1.5
self.wait(1)
pointer_value -= 4
self.wait(2)
```

**Review by RickyC0626 — APPROVED** [review]
Looks good 👍🏻

File: manim/mobject/value_tracker.py (hunk 1 above)
Comment (kolibril13) [reply manim/mobject/value_tracker.py:47]: As pointer_value += 1.5 is a sudden jump, I think waiting  `self.wait(0.5)`is enough.
We also should keep in mind when making the docs example, that each additional frame will increase the overall building time of the documentation, as every time we make a commit, the whole docs (including all examples) are rendered from scratch.

Comment (behackl) [reply manim/mobject/value_tracker.py:47]: I don't think adding longer `wait`s increases the doc build time substantially (if it increases them at all).

Comment (kolibril13) [reply manim/mobject/value_tracker.py:47]: Okay.
I actually prefer the shorter time: https://manimce--1246.org.readthedocs.build/en/1246/reference/manim.mobject.value_tracker.ValueTracker.html?highlight=valuetracker#valuetrackerexample
But that is personal taste.
@RickyC0626 : If you think it is a lot better with longer time, feel free to push a commit, but I prefer the current solution.

File: manim/mobject/value_tracker.py [hunk 2]
```diff
@@ -48,19 +53,28 @@ def __init__(self, value=0, **kwargs):
...
+        """Sets a new value (float, int) to the ValueTracker"""
         self.points[0, 0] = value
         return self
 
     def increment_value(self, d_value):
+        """Increments (adds) a value (float, int) to the ValueTracker"""
         self.set_value(self.get_value() + d_value)
 
     def __iadd__(self, d_value):
+        """adds `+=` syntax to increment the value of the ValueTracker"""
```
Comment (naveen521kk) [inline manim/mobject/value_tracker.py:69]: it's rst, use double ` for code and not single.

**Pull request merged**